index.bs

```diff
@@ -128,7 +140,25 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
     </pre>
 
     <div dfn-type="method" dfn-for="IntersectionObserver">
-        : <dfn>observe()</dfn>
+        : <dfn>Constructor(callback, options)</dfn>
+        ::
+            <ol>
+                <li>Let <i>this</i> be a new {{IntersectionObserver}}
+                    object</li>
+                <li> Set <i>this</i>'s internal {{[[callback]]}} slot to
+                    <i>callback</i>.</li>
+                <li> Set <i>this</i>'s internal {{[[root]]}} slot to
+                    <i>options</i>.{{root}}</li>
+                <li> Set <i>this</i>'s internal {{[[rootMargin]]}} slot to
+                    <i>options</i>.{{rootMargin}}</li>
+                <li> Set <i>this</i>'s internal {{[[threshold]]}} slot to
+                    <i>options</i>.{{threshold}}</li>
+                <li>Append <i>this</i> to the
+                    <a>unit of related similar-origin browsing contexts</a>'s
+                    internal <a>\[[IntersectionObservers]]</a> slot.</li>
```

Comment:
IntersectionObservers was the one private slot I couldn't figure out how to add as an IDL and had to do it as a DFN. Because it's associated with a 'browsing context' which isn't really an object, I couldn't figure out how to make it a property. So that's why I always used an \<a> to link to it instead of a {{ }}. Any suggestions?